portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,17 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="bg"></div>
+      <div id="home_content">
```

Comment:
Hyphen instead of underscore: https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters

Reply:
Add hyphen

Reply:
add hyphen

Reply:
add hyphen